src/Adapters/Redis/MetricDto.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lamoda\Metric\Adapters\Redis;
+
+final class MetricDto
```

Comment:
Pls mark this class as `@internal`. And also check all classes and interfaces that are not supposed to be used directly by end user and mark them all to

Reply:
Done